### ISSUE TITLE
Installation failure #1

### ISSUE THREAD

**Issue by rageworx** [rageworx/homebrew-listusb#1]
There's error occurs while try to install as ...
```
Error: listusb: /usr/local/Homebrew/Library/Taps/rageworx/homebrew-listusb/Formula/listusb.rb:12: syntax error, unexpected tIDENTIFIER, expecting end
      bin.install "listusb"
                   ^~~~~~~
/usr/local/Homebrew/Library/Taps/rageworx/homebrew-listusb/Formula/listusb.rb:14: unterminated string meets end of file
/usr/local/Homebrew/Library/Taps/rageworx/homebrew-listusb/Formula/listusb.rb:14: syntax error, unexpected end-of-input, expecting end
```

**Comment by rageworx** [comment]
listusb.rb fixed typoerrors, but still error occurs,
```
Error: No available formula with the name "listusb". Did you mean lsusb or libgusb?
In formula file: /usr/local/Homebrew/Library/Taps/rageworx/homebrew-listusb/Formula/listusb.rb
Expected to find class Listusb, but only found: Rsdic.
```

**Comment by rageworx** [comment]
There's error found class name 'Rsdic', fixed to 'listusb'.

**Comment by rageworx** [comment]
Duuno what meaning of this,
```
Error: listusb: /usr/local/Homebrew/Library/Taps/rageworx/homebrew-listusb/Formula/listusb.rb:4: class/module name must be CONSTANT
class listusb < Formula
      ^~~~~~~
```

**Comment by rageworx** [comment]
Class name CASE SENSETIVE !
```
Expected to find class Listusb, but only found: ListUSB.
```

**Comment by rageworx** [comment]
Solved this issue with v0.1.0.5 tagging.
